apps/meteor/app/statistics/server/lib/statistics.ts

```diff
@@ -146,14 +146,14 @@ export const statistics = {
 
 		// Number of departments
 		statsPms.push(
-			LivechatDepartment.col.count().then((count) => {
+			LivechatDepartment.col.estimatedDocumentCount().then((count) => {
```

Comment:
`estimatedDocumentCount` could be a `BaseRaw` method, wdyt?

Reply:
🤔 that works, let me add it